webapp/src/octoClient.ts

```diff
@@ -69,11 +71,11 @@ class OctoClient {
     async login(username: string, password: string): Promise<boolean> {
         const path = '/api/v2/login'
         const body = JSON.stringify({username, password, type: 'normal'})
-        const response = await fetch(this.getBaseURL() + path, {
+        const response = await fetch(this.getBaseURL() + path, Client4.getOptions({
```

Comment:
How can we guarantee that future calls here properly use `Client4.getOptions`? We should consider creating a base fetch call that makese sure to pass through the `options` through the client4 as Playbooks is doing: https://github.com/mattermost/mattermost-plugin-playbooks/blob/4d758cb0e2288cd9e2097c43c32f1c20a8bb5108/webapp/src/client.ts#L724-L725

Reply:
I've created a ticket here - https://mattermost.atlassian.net/browse/MM-65168